content/en/contribute/code/core/deploy-on-eks.md

```diff
@@ -0,0 +1,132 @@
+2. Once the ticket in step one is complete, follow the [CLI setup guide](https://github.com/medic/medic-infrastructure/blob/master/terraform/aws/dev/eks/access/README.md).
+
+**NB** - Security key (e.g. Yubikey) users need to add a TOTP MFA (Time-based, One-Time Password Multi-Factor Authentication) too! CLI requires the TOTP values (6-digit number) and security keys are not supported. Security keys can only be used on web logins.
+
+### First time setup
+
+These steps only need to be run once!
+
+After you have created a ticket per "Request permission" above, you should get a link to sign up for AWS.  Click the link and:
+
+1. Create new password ensure it's 10+ characters including one alpha (`a-z`) and one special (``~!@#$%^&*_-+=`|\(){}[]:;"'<>,.?/``) character.
+2. Setup MFA. In top-right corner of browser, there is a drop-down menu with your `username @ medic`. Click that and then on "My Security Credentials"
+3. Assign an MFA device and give it the **same name** as your username: In AWS web GUI, click your name in upper right:
+   1. Security Credentials 
+   2. scroll down to "Multi-factor authentication (MFA)" 
+   3. click "Assign MFA device" 
+   4. enter a "Device name" (should match username) 
+   5. "Select MFA device" that you're using
+4. Create Access Keys for Command Line Interface: In AWS web GUI, click your name in upper right -> Security Credentials -> scroll down to "Access keys" -> click "Create access key" -> for use case choose "Command Line Interface" -> click "Next" -> enter description and click "Create access key"
+5. Run `aws configure` and place appropriate access keys during prompts. Use `eu-west-2` region. It should look like this:
+
+   ```
+   $ aws configure
+
+   AWS Access Key ID [None]: <ACCESS-KEY-HERE>
+   AWS Secret Access Key [None]: <SECRET-HERE>
+   Default region name [None]: eu-west-2
+   Default output format [None]:
+   ```
+6. Run the Update Kubeconfig command, assuming username is `mrjones` and namespace is `mrjones-dev` - be sure to place these with yours: `aws eks update-kubeconfig --name mrjones-dev --profile mrjones --region eu-west-2`
+
+## Autocomplete
+
+Both `helm` and `kubectl` have autocomplete libraries you can add to your shell. For power users and beginners alike, this adds a huge amount of discoverability. This guide covers `zsh`, but `bash`, `fish` and `powershell` are supported as well:
+
+1. Output `zsh` shell's setup to a file: 
+   ```shell
+   kubectl completion zsh > zsh.kubectl.autocomplete
+   helm completion zsh > zsh.helm.autocomplete
+   ```
+  
+2. Then source the two new files:
+   ```shell
+   source zsh.kubectl.autocomplete
+   source zsh.helm.autocomplete
+   ```
+3. See [helm](https://helm.sh/docs/helm/helm_completion_bash/) and [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#enable-shell-autocompletion) instructions for automatically loading these on every new session.
```

Comment:
Does it make sense to move this section to be a subsection under **Command Line** above?

Reply:
yup - agreed!  also I can tighten them by combining the steps - see c4f8454d